src/directives/resizeable.directive.ts

```diff
@@ -29,16 +30,17 @@ export class ResizeableDirective implements OnDestroy, AfterViewInit {
 
   ngAfterViewInit(): void {
     const renderer2 = this.renderer;
-    const node = renderer2.createElement('span');
+    this.createdNode = renderer2.createElement('span');
     if (this.resizeEnabled) {
-      renderer2.addClass(node, 'resize-handle');
+      renderer2.addClass(this.createdNode, 'resize-handle');
     } else {
-      renderer2.addClass(node, 'resize-handle--not-resizable');
+      renderer2.addClass(this.createdNode, 'resize-handle--not-resizable');
     }
-    renderer2.appendChild(this.element, node);
+    renderer2.appendChild(this.element, this.createdNode);
   }
 
   ngOnDestroy(): void {
+    this.renderer.destroyNode(this.createdNode);
```

Comment:
*destroyNode* is a debug only function, you should be using *removeChild* instead I guess.